### ISSUE TITLE
The safeHtml property returns String instead of html code

### ISSUE THREAD

**Issue by jpoiri** [benjamminf/craft-embedded-assets#24]
Hi there,

When I try to add the following code in my template to embed a youtube video, the iframe is just outputting the code as text on the screen instead of adding a iframe to the DOM

     {% set embeddedAsset = craft.embeddedAssets.fromAsset(block.video.first()) %}
     {% if embeddedAsset %}
            {{ embeddedAsset.safeHtml }}
     {% endif %}`

![embedded](https://cloud.githubusercontent.com/assets/6857740/17146314/594a21bc-5334-11e6-9ce4-aac20f41e5d3.png)

**Comment by benjamminf** [comment]
Hm, interesting. This might have happened to me before, so as a safe-guard you can use the `raw` filter to prevent this from happening:

```twig
{{ embeddedAsset.safeHtml|raw }}
```

Outputting HTML strings is something that's always seemed to behave randomly.

**Comment by chrissam42** [comment]
For some reason the `|raw` filter doesn't seem to work inside Twig's ternary operator (`{{ x ? y : z}}`). I thought it might be a Twig or Craft bug, but I tried a number of simple tests using strings and object properties containing HTML, and they all worked as expected. But on these EmbeddedAssetModel `html` and `safeHtml` properties, `|raw` has no effect.

```
{% set embedded = craft.embeddedAssets.fromAsset(asset) %}

{# doesn't work, always outputs entity-encoded HTML #}
{{ embedded ? embedded.html|raw : '...' }}

{# works as expected #}
{% if embedded %}{{embedded.html|raw}}{% else %}...{% endif %}
```

I tried adding parentheses to avoid parsing confusion, but no help.

I'm running Craft 2.6.3013 and Twig 1.35.0.
